### DISCUSSION TITLE
[BP#4] Préférer la saisie assistée à l'autocomplétions

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#201] [Bonnes pratiques] [answered]
Je suis assez étonné de la mesure de la règle de validation, 20% des champs de saisie me semble très élevé (i.e quasiment toujours ok).

Sinon je ne trouve pas très explicite la notion de saisie assistée "ensembles d'informations et d'indices" (ce qui est entre parenthèses ne m'aide pas) et l'exemple...quand je pense autocompletion, je pense plutôt recherche que donnée formatée par exemple.

Enfin, il y a quelques fautes : dans le titre, complétion vs complètement...

**Comment by rlemaire** [comment]
+1 pour améliorer la description.
On peut aussi parler des améliorations de l'autocomplétion : délai entre la saisie et la requête, nombre de caractère minimal, mise en cache, contextualisation, ...

**Comment by ACTLEM** [comment, marked as answer]
Ok avec les remarques.

Issue créée: https://github.com/cnumr/best-practices/issues/281
